test.sh

```diff
@@ -65,6 +65,10 @@ function docker_compose_run {
     invoke_docker_compose run --rm --user `id -u`:`id -g` "$@"
 }
 
+function docker_compose_run_fe {
```

Comment:
can be removed now that we merged #2045?

Reply:
Done.